Makefile

```diff
@@ -171,19 +172,31 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: ko-build-local
 ko-build-local: test ## Build local container image with ko
 	$(shell (docker version | { ! grep -qi podman; } ) || echo "DOCKER_HOST=unix://$$HOME/.local/share/containers/podman/machine/qemu/podman.sock ") KO_DOCKER_REPO=ko.local ko build -B ./cmd/${CMD_NAME} -t ${IMAGE_TAG} --platform linux/${ARCH}
-	docker tag ko.local/${CMD_NAME}:${IMAGE_TAG} ${IMG}
+	docker tag ko.local/${CMD_NAME}:${IMAGE_TAG} ${IMAGE_REPO}:${IMAGE_TAG}
 
 # this is used for local testing 
 .PHONY: kind-load-image
 kind-load-image: 
-	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${IMG}
+	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${IMAGE_REPO}:${IMAGE_TAG}
 
 .PHONY: chart
 chart: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${IMG} | sed 's/\(:.*\)v/\1/')
 	$(KUSTOMIZE) build config/default > chart/templates/controller-manager.yaml
 	scripts/add-helm-code.sh add
 
+.PHONY: local-chart
+local-chart: manifests kustomize
+ifeq (1,$(shell (git status | grep "config/manager/kustomization.yaml" | wc -l)))
+	@echo 'ERROR: config/manager/kustomization.yaml is already checked out!'
```

Comment:
I think that we need `make local-chart` to fail in this case.

Would it work to put the command `exit 1` in the script in this case? BTW, if so, then we could replace the Makefile "if" statement with a bash "if" statement.

Reply:
Good point.  I added an exit.  My bash-fu in make is not strong enough to switch to bash, so I used the native makefile if statements.